custom/icds_reports/static/js/angular-services/locations.service.js

```diff
@@ -117,11 +117,7 @@ window.angular.module('icdsApp').factory('locationsService', ['$http', '$locatio
                 if (!selectedLocation || selectedLocation.location_id === 'all') {
                     return [];
                 }
-                return _.sortBy(
```

Comment:
will this not stop sorting all together? we want to sort actual location names

Reply:
they are getting sorted in the backend. Don't think elements will be added to it runtime in the UI to sort